docs/ContainerizedSetup.asciidoc

```diff
@@ -109,13 +102,82 @@ next two steps, you will set an OpenID provider (if necessary), create the API
+command substituting `KEY` and `SECRET` with the generated values:
+
+    docker exec -it openqa_data /scripts/client-conf set -l KEY SECRET
+
+=== Run the worker container
+
+    docker run -d -h openqa_worker_1 --name openqa_worker_1 --link openqa_webui:openqa_webui --volumes-from openqa_data --privileged fedoraqa/openqa_worker
+
+Check whether the worker connected in the web UI's administration interface.
+
+To add more workers, increase the number that is used in hostname and
+container name, so to add worker 2 use:
+
+    docker run -d -h openqa_worker_2 --name openqa_worker_2 --link openqa_webui:openqa_webui --volumes-from openqa_data --privileged fedoraqa/openqa_worker
+
+=== Enable services
+
+Some systemd services are provided to start up the containers, so you do not
+have to keep doing it manually. To install and enable them:
+
+    sudo cp systemd/*.service /etc/systemd/system
+    sudo systemctl daemon-reload
+    sudo systemctl enable openqa-data.service
+    sudo systemctl enable openqa-webui.service
+    sudo systemctl enable openqa-worker@1.service
+
+Of course, if you set up two workers, also do `sudo systemctl enable
+openqa-worker@2.service`, and so on.
+
+=== Get tests, ISOs and create disks
+
+You have to put your tests under `data/tests` directory and ISOs under
+`data/factory/iso` directory. For testing Fedora, run:
+
+    git clone https://bitbucket.org/rajcze/openqa_fedora data/tests/fedora
+    wget https://dl.fedoraproject.org/pub/alt/stage/22_Beta_RC3/Server/x86_64/iso/Fedora-Server-netinst-x86_64-22_Beta.iso -O data/factory/iso/Fedora-Server-netinst-x86_64-22_Beta_RC3.iso
+
+And set permissions, so any user can read/write the data:
+
+    chmod -R 777 data
+
+This step is unfortunately necessary because Docker
+https://github.com/docker/docker/issues/7198[can not mount a volume with specific user ownership]
+in container, so ownership of mounted folders (uid and gid) is the same as on
+your host system (presumably 1000:1000 which maps into nonexistent user in all
+of the containers).
+
+If you wish to keep the tests (for example) separate from the shared
+directory, for any reason (we do, in our development scenario) refer to the
+[Developing tests with Container setup] section at the end of this document.
+
+Populate the openQA database:
+
+    docker exec openqa_webui /var/lib/openqa/tests/fedora/templates
+
+Create all necessary disk images:
+
+    cd data/factory/hdd && createhdds.sh VERSION
+
+where `VERSION` is the current stable Fedora version (its images will be
+created for upgrade tests) and createhdds.sh is in `openqa_fedora_tools`
+repository in `/tools` directory. Note that you have to have
+`libguestfs-tools` and `libguestfs-xfs` installed.
+
+== Setup with openSUSE-based images and docker-compose
+
+=== Run the data and web UI containers in HA mode
 
     # To create the containers
     # in the directory openQA/docker/webui execute:
     docker-compose up -d
```

Comment:
Here and related with I mentioned previously, docker-compose up is going to build the images if don't exist. Maybe is interesting to comment that too because simplifies all the setup

Reply:
That would be good to know. I suppose I'll have to try it out myself to see how it is actually supposed to be used *without* building any images before manually.